Feliz/Properties.fs

```diff
@@ -365,10 +365,18 @@ type prop =
     ///
     /// <canvas>, <embed>, <iframe>, <img>, <input>, <object>, <video>
     static member inline height(value: int) = Interop.mkAttr "height" value
+    /// Specifies the height of elements listed here. For all other elements, use the CSS height property.
+    ///
+    /// <canvas>, <embed>, <iframe>, <img>, <input>, <object>, <video>
```

Comment:
I think you have to add backticks around these tags, otherwise they won't display properly. e.g.

```f#
/// `<canvas>`, `<embed>`, ...
```

I see this is an issue with the existing overloads, too.